Build Failure on refs/heads/main

@gemini-code-assist /jules debug this build error

**Log:**
```
eDebugGeneratedProguardFiles
> Task :feature:ar:mergeDebugConsumerProguardFiles
> Task :feature:ar:mergeDebugJavaResource
> Task :feature:ar:syncDebugLibJars FROM-CACHE
> Task :feature:ar:bundleDebugAar
> Task :feature:ar:assembleDebug
> Task :opencv:bundleDebugAar
> Task :opencv:assembleDebug
> Task :core:nativebridge:buildCMakeDebug[arm64-v8a]
> Task :core:nativebridge:configureCMakeDebug[armeabi-v7a]
> Task :feature:editor:kspDebugKotlin
> Task :core:nativebridge:buildCMakeDebug[armeabi-v7a]
> Task :core:nativebridge:mergeDebugJniLibFolders
> Task :core:nativebridge:checkDebugAarMetadata
> Task :core:nativebridge:extractDebugAnnotations FROM-CACHE
> Task :core:nativebridge:extractDeepLinksForAarDebug FROM-CACHE
> Task :core:nativebridge:mergeDebugGeneratedProguardFiles
> Task :core:nativebridge:mergeDebugConsumerProguardFiles
> Task :core:nativebridge:prepareDebugArtProfile
> Task :core:nativebridge:prepareLintJarForPublish
> Task :core:nativebridge:mergeDebugJavaResource
> Task :core:nativebridge:syncDebugLibJars FROM-CACHE
> Task :core:nativebridge:mergeDebugNativeLibs
> Task :core:nativebridge:stripDebugDebugSymbols
> Task :core:nativebridge:copyDebugJniLibsProjectOnly
> Task :feature:editor:extractDebugAnnotations
> Task :core:nativebridge:copyDebugJniLibsProjectAndLocalJars
> Task :app:mergeDebugNativeLibs
> Task :core:nativebridge:bundleDebugAar
> Task :core:nativebridge:assembleDebug
> Task :app:stripDebugDebugSymbols

> Task :feature:editor:compileDebugKotlin
w: ATTENTION!
This build uses unsafe internal compiler arguments:

-XXLanguage:+PropertyParamAnnotationDefaultTargetMode

This mode is not recommended for production use,
as no stability/compatibility guarantees are given on
compiler or generated code. Use it at your own risk!


> Task :feature:editor:compileDebugJavaWithJavac
> Task :feature:editor:processDebugJavaRes
> Task :feature:editor:mergeDebugJavaResource
> Task :feature:editor:transformDebugClassesWithAsm
> Task :feature:editor:bundleLibCompileToJarDebug
> Task :feature:editor:bundleLibRuntimeToJarDebug
> Task :feature:editor:mergeDebugGeneratedProguardFiles
> Task :feature:editor:mergeDebugConsumerProguardFiles
> Task :feature:editor:bundleLibRuntimeToDirDebug
> Task :feature:editor:syncDebugLibJars
> Task :feature:editor:bundleDebugAar
> Task :feature:editor:assembleDebug
> Task :app:kspDebugKotlin
> Task :app:mergeLibDexDebug

> Task :app:compileDebugKotlin FAILED
e: file:///home/runner/work/GraffitiXR/GraffitiXR/app/src/main/java/com/hereliesaz/graffitixr/MainActivity.kt:579:37 Unresolved reference 'OverlayScreen'.
e: file:///home/runner/work/GraffitiXR/GraffitiXR/app/src/main/java/com/hereliesaz/graffitixr/MainActivity.kt:583:37 Unresolved reference 'MockupScreen'.
gradle/actions: Writing build results to /home/runner/work/_temp/.gradle-actions/build-results/gradle-build-1776535876973.json

[Incubating] Problems report is available at: file:///home/runner/work/GraffitiXR/GraffitiXR/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:compileDebugKotlin'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.btapi.BuildToolsApiCompilationWork
   > Compilation error. See log for more details

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights from a Build Scan (powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.4.0-rc-1/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 1m 24s
331 actionable tasks: 185 executed, 146 from cache

```

--- a/app/src/main/java/com/hereliesaz/graffitixr/MainActivity.kt
+++ b/app/src/main/java/com/hereliesaz/graffitixr/MainActivity.kt
@@ -101,6 +101,8 @@ import com.hereliesaz.graffitixr.feature.dashboard.SettingsScreen
 import com.hereliesaz.graffitixr.feature.dashboard.SettingsViewModel
 import com.hereliesaz.graffitixr.feature.editor.EditorUi
 import com.hereliesaz.graffitixr.feature.editor.EditorViewModel
+import com.hereliesaz.graffitixr.feature.editor.MockupScreen
+import com.hereliesaz.graffitixr.feature.editor.OverlayScreen
 import com.hereliesaz.graffitixr.feature.editor.TraceScreen
 import com.hereliesaz.graffitixr.data.OnboardingManager
 import com.hereliesaz.graffitixr.feature.editor.util.ImageProcessor as EditorImageProcessor

